test/test_helper.js

```diff
@@ -453,7 +453,7 @@ class TestHelper {
         this.basicGroup = this.fakeGroupWithId();
     }
 
-    initBasic = async (client4 = this.createClient4()) => {
+    initBasic = (client4 = this.createClient4()) => {
```

Comment:
Oh, I see we've actually stripped this here -- that makes more sense now!

Reply:
Yeah, I don't know why it was even there to begin with. It was probably something we needed at some point, but never ended up using after that. We have had a tendency to overuse async/await frequently in this code.